sunbeam/sunbeam.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef SUNBEAM_HPP
+#define SUNBEAM_HPP
+
+#include <boost/python.hpp>
+
+#include "converters.hpp"
+
+namespace Opm { }
+namespace py = boost::python;
+
+using namespace Opm;
+using ref = py::return_internal_reference<>;
+using copy = py::return_value_policy< py::copy_const_reference >;
+
+namespace sunbeam {
+
+void export_Completion();
+void export_Deck();
+void export_DeckKeyword();
+void export_Eclipse3DProperties();
+void export_EclipseConfig();
+void export_EclipseGrid();
+void export_EclipseState();
+void export_Group();
+void export_Parser();
+void export_Schedule();
+void export_TableManager();
+void export_Well();
+void export_GroupTree();
```

Comment:
This is a list of exports on *very* different levels of abstraction; i.e. `Completion` is quite minor detail - essentially a struct with a handfull of members, whereas e.g. `Schedule` is a large objects with large subobjects with subobjects which finally contain a completion, i.e. could these `export_xxxx` statements be grouped and moved e.g. like this:

```C++
// sunbeam.hpp

void export_Deck();
void export_EclipseState();
void export Parser();
 ```
```C++
// eclipsestate.hpp

void export_Schedule();
void export_EclipseGrid();
void export_TableManager();
```
And so on?

Reply:
I agree, but this change were made in #39 which is now merged into master. I have now rebased with lastest master.